templates/common-steps.tpl

```diff
@@ -0,0 +1,23 @@
+{{- define "common-steps" -}}
+- name: prepare
+  image: {{ .Values.images.gitInit }}
+  workingDir: $(workspaces.source.path)
+  command:
+    - /scripts/prepare.sh
+  volumeMounts:
+    - name: scripts-dir
+      mountPath: /scripts
+    - name: user-home
+      mountPath: "$(params.USER_HOME)"
```

Comment:
I think the `prepare` step is shared between the two Tasks, `git-clone` and `git-cli`.

Reply:
I didnt get you, prepare step is shared thats true, so im keeping it in the steps template, is there any other way you are trying to say?